packages/react-components/src/overlay/src/useTriggerWidth.ts

```diff
@@ -1,8 +1,12 @@
 import { arrayify, useEventCallback, useResizeObserver } from "../../shared";
 import { useState } from "react";
 
-export function useTriggerWidth(triggerElement, { isDisabled } = {}) {
-    const [triggerWidth, setTriggerWidth] = useState();
+export interface UseTriggerWidthProps {
+    isDisabled?: boolean;
+}
+
+export function useTriggerWidth(triggerElement: Element, { isDisabled }: UseTriggerWidthProps = {}) {
```

Comment:
Not sure about this `UseTriggerWidthProps`  interface? If you define an interface for `useTriggerWidth` it should contains all props not only optionnal props.